system/file.go

```diff
@@ -214,3 +216,23 @@ func realPath(path string) (string, error) {
 
 	return realPath, err
 }
+
+func (f *DefFile) Md5() (string, error) {
```

Comment:
Ah yes... That's clear after another look at it 👍 this was a super quick addition initially.